docs/tutorials/multimodal/multimodal-faq.md

```diff
@@ -0,0 +1,30 @@
+# FAQ
+:label:`sec_multimodal_faq`
+
+
+### There is no internet access in my deployment environment. What should I do? 
+
+When you have trained the predictor, try to save it with
+
+```python
+predictor.save(SAVE_PATH, standalone=True)
+```
+
+Afterwards, the following `.load()` call can happen without internet access:
+
+```python
+from autogluon.multimodal import MultiModalPredictor
+
+predictor = MultiModalPredictor.load(SAVE_PATH)
+```
+
+
+### Do I need to preprocess my text or image data before using AutoGluon Multimodal?
+
+Usually you do not need to preprocess the text / image data. AutoGluon Multimodal has built-in 
+support of text / image preprocessing. However, this won't block you from appending custom preprocessing logic before 
+feeding in the dataframe to AutoGluon Multimodal.
```

Comment:
`However, this won't block you from appending custom preprocessing logic before 
feeding in the dataframe to AutoGluon Multimodal.` Just want to confirm, if user already preprocess the image (e.g., minus 255, divided by customized mean, or data augmentation), will AutoGluon do another round of preprocessing and data augmentation?

Reply:
we will do another round of normalization. As in https://github.com/autogluon/autogluon/blob/3f0e36d9ee14dcd11f71d819fdf8be0b40b837da/multimodal/src/autogluon/multimodal/data/process_image.py#L313

Reply:
I can add a flag to disable normalization in a later PR. I think this is a good point.